rke/templates/templates.go

```diff
@@ -268,11 +272,12 @@ func getTemplates() map[string]string {
 		calicov3225:          CalicoTemplateV3_22_5,
 		calicov3241:          CalicoTemplateV3_24_1,
 
-		flannelv115:  FlannelTemplateV115,
-		flannelv116:  FlannelTemplateV116,
-		flannelv0140: FlannelTemplateV0_14_0,
-		flannelv0192: FlannelTemplateV0_19_2,
-		flannelv18:   FlannelTemplate,
+		flannelv115:          FlannelTemplateV115,
+		flannelv116:          FlannelTemplateV116,
+		flannelv0140:         FlannelTemplateV0_14_0,
+		flannelv0140Rancher2: flannelv0140Rancher2,
```

Comment:
I'm pretty sure the value should be `FlannelTemplateV0_14_0Rancher2` here.

Reply:
updated